scripts/events_module/relationship/pregnancy_events.py

```diff
@@ -947,22 +946,18 @@ def get_kits(
                     change_relationship_values(
                         cats_from=[kit],
                         cats_to=[parent],
-                        platonic_like=kit_to_parent["platonic"],
-                        dislike=kit_to_parent["dislike"],
-                        admiration=kit_to_parent["admiration"],
-                        comfortable=kit_to_parent["comfortable"],
-                        jealousy=kit_to_parent["jealousy"],
-                        trust=kit_to_parent["trust"],
+                        like=kit_to_parent[RelValue.LIKE],
+                        respect=kit_to_parent[RelValue.RESPECT],
+                        comfort=kit_to_parent[RelValue.COMFORT],
+                        trust=kit_to_parent[RelValue.TRUST],
                     )
                     change_relationship_values(
                         cats_from=[parent],
                         cats_to=[kit],
-                        platonic_like=parent_to_kit["platonic"],
-                        dislike=parent_to_kit["dislike"],
-                        admiration=parent_to_kit["admiration"],
-                        comfortable=parent_to_kit["comfortable"],
-                        jealousy=parent_to_kit["jealousy"],
-                        trust=parent_to_kit["trust"],
+                        like=parent_to_kit[RelValue.LIKE],
+                        respect=parent_to_kit[RelValue.RESPECT],
+                        comfort=parent_to_kit[RelValue.COMFORT],
+                        trust=parent_to_kit[RelValue.TRUST],
```

Comment:
```suggestion
                        **parent_to_kit,
```

should replace lines 957-960